pkg/convert/xliff12_test.go

```diff
@@ -101,6 +103,33 @@ func Test_FromXliff12(t *testing.T) {
 			data:     randXliff12(nonOriginalTranslation),
 			expected: nonOriginalTranslation,
 		},
+		{
```

Comment:
Maybe you could supplement test with more cases?
Example: 
Message: `Order #{Id} has been canceled for {ClientName}`,